src/qt/bitcoingui.cpp

```diff
@@ -1119,16 +1120,21 @@ void BitcoinGUI::detectShutdown()
     }
 }
 
-void BitcoinGUI::showProgress(const QString &title, int nProgress)
+void BitcoinGUI::showProgress(const QString &title, int nProgress, const std::function<void(void)> &cancel)
```

Comment:
`&` are next to the type. Same for remaining changes.

Reply:
Done